deploy_liteserv.py

```diff
@@ -0,0 +1,27 @@
+import sys
+import subprocess
+import time
+
+from optparse import OptionParser
+
+# jython imports
+from com.android.monkeyrunner import MonkeyRunner, MonkeyDevice
```

Comment:
@andrei-baranouski `monkeyrunner` is a jython based commandline tool that is shipped with the Android SDK, it bundles all of the necessary dependencies already.